### PR TITLE
Fix for Delete Chain Route

### PR DESCRIPTION
**Pull request by alexyoung23j**
## Description

Some new tables required additional DELETE statements for the entire delete chain route to work.

## Motivation and Context

BD wants to delete communities!

## Does this PR affect any server routes?
- [ ] yes
- [ ] no

## Have proper tags been added (for bug, enhancement, breaking change)?
- [ ] yes

### CODE REVIEW

File: packages/commonwealth/server/routes/deleteChain.ts [hunk 1]
```diff
@@ -251,6 +252,14 @@ const deleteChain = async (
     );
 
     // TODO: Remove this once we figure out a better way to relate addresses across many chains (token communities)
+    await models.sequelize.query(
```
Comment (jnaviask) [inline packages/commonwealth/server/routes/deleteChain.ts:255]: Can we do this all with Sequelize instead of raw SQL?

Comment (alexyoung23j) [reply packages/commonwealth/server/routes/deleteChain.ts:255]: Want the whole route changed to match or just these?

Comment (jnaviask) [reply packages/commonwealth/server/routes/deleteChain.ts:255]: Ideally the whole route.

Comment (alexyoung23j) [reply packages/commonwealth/server/routes/deleteChain.ts:255]: okay

**Pull request merged**